torch/csrc/jit/script/init.cpp

```diff
@@ -115,7 +115,10 @@ struct MethodValue : public SugaredValue {
     }
     ensureSizeMatches(loc, method.num_inputs(), inputs.size(), "inputs");
     auto outputs = caller.emit_call_to(method, inputs);
-    ensureSizeMatches(loc, outputs.size(), n_outputs, "outputs");
+    // Allow for tuples. TODO: starred exprs?
+    if (n_outputs != 1 && n_outputs != -1) {
```

Comment:
Why the `n_outputs != 1`? Do we just assign the outputs as a tuple in that case?

Reply:
Yeah